xbmc/interfaces/legacy/ListItem.h

```diff
@@ -640,7 +640,7 @@ namespace XBMCAddon
       /// | developer     | string (Square)
       /// | overview      | string (Long Description)
       /// | year          | integer (1980)
-      /// | emulator      | string (game.libretro.fceumm)
+      /// | gameclient    | string (game.libretro.fceumm)
```

Comment:
Please also add
`@python_v18 Added new  types **game** and associated infolabels.`
or something few lines lower